Doc/whatsnew/3.0.rst

```diff
@@ -53,9 +53,9 @@
 
 This article explains the new features in Python 3.0, compared to 2.6.
 Python 3.0, also known as "Python 3000" or "Py3K", is the first ever
-*intentionally backwards incompatible* Python release.  There are more
-changes than in a typical release, and more that are important for all
-Python users.  Nevertheless, after digesting the changes, you'll find
+*intentionally backwards incompatible* Python release. Python 3.0 was released on 3 Dec 2008.
```

Comment:
I think that the date formatting should be (somewhat) consistent, at least for the whatsnew pages changed in this PR. Looking at [3.9's whatsnew](https://docs.python.org/3/whatsnew/3.9.html), it has it as 'October 5th, 2020', and 3.7 has it as 'June 27, 2018', so `month, day, year` seems like a good enough standard to go off in this case.

Reply:
I agree with date formatting consistency. Made adjustments to all versions.